src/etc/diffuserc

```diff
@@ -2,4 +2,4 @@
 #
 # Copyright (C) 2006-2009 Derrick Moser <derrick_moser@yahoo.com>
 
-import ../usr/share/diffuse/syntax/*.syntax
+import /usr/local/share/diffuse/syntax/*.syntax
```

Comment:
I'm a little unsure about this change, before we were looking for a local file and now we're looking for an absolute file. What if the installation path is not /usr/local?

Reply:
yeah, this change is almost certainly not the right answer, I threw it in to start the conversation as it fixed problems for me when the install path was /usr/local. I suppose the real problem is that .. is somehow not referring to the right relative path.

Reply:
I merged your first commit with #29, but I'm unsure about the second one. That would need more tests.

Reply:
Hi @dscherger !

As you might have seen, I've spent a bit more time recently with the Diffuse source code. I can say now that this change will break installation, at least on Linux.

If you want to know more, the source of the problem is in `install.py` at line 249, which looks like that once a bit reformatted:
```python
processFiles(
    install,
    os.path.join(destdir, examplesdir[1:]),
    'src/etc/',
    {
        'diffuserc': [
            (b'/etc/', sysconfdir.encode()),
            (b'../usr', relpath(sysconfdir, prefix).encode())
        ]
    })
```

As ugly as it is, this code takes `src/etc/diffuserc` and replaces `/etc/` by `sysconfdir.encode()`, and `../usr` by `relpath(sysconfdir, prefix).encode()`.

This means that your change will break the text replacement since "../usr" won't exist any more.

I think the solution is a bit more complex than that and lies in #52.

Is it ok with you if I close your PR?

Cheers!

Reply:
Yes by all means, go ahead and close it. I was just bashing on things trying to get something working on OSX. Your new ideas sound good.